RefactoringJava/src/main/java/com/myheincompany/refactroingJava/dataStore/ProgrammerBuilder.java

```diff
@@ -1,48 +0,0 @@
-/*
- * To change this license header, choose License Headers in Project Properties.
- * To change this template file, choose Tools | Templates
- * and open the template in the editor.
- */
-package com.myheincompany.refactroingJava.dataStore;
-
-
-public class ProgrammerBuilder {
-    private String name = "Hans";
```

Comment:
don't set the variables with values, because now every new programmer has the name Hans Peter ( if you dont set it before calling createProgrammer)